src/models/hooks/genericLink.js

```diff
@@ -41,8 +41,8 @@ const syncLink = async (
   // Generate a unique semaphore key based on the model name and entity ID
   const semaphoreKey = `${model.tableName}_${entityId}`;
   // Acquire a lock to ensure only one operation is performed on this entity at a time
-  await semaphore.acquire(semaphoreKey);
```

Comment:
Looks like a newline migrating from below to above the code that goes with the comment